simulation_lab/BaseTestCaseView.py

```diff
@@ -11,6 +11,9 @@ def assertMessageLevel(self, response, messageLevel):
         assert messageLevel in [x.tags for x in response.context['messages']],f"{messageLevel} banner was not provided"
 
     def createLoggedInUser(self, isAdmin):
-        # TODO: Implement logic for admin
+        if isAdmin:
+            User.objects.create_superuser("test@test.com", "Test1234")
+        else:
+            User.objects.create_user("test@test.com", "Test1234")
 
-        self.client.force_login(User.objects.create_user("test@a.com", "test"))
+        #self.client.force_login(User.objects.create_user("test@a.com", "test"))
```

Comment:
You still need self.client.force_login(). Otherwise you are just creating a user and not logging them  in.